tests/emukit/quadrature/test_square_transform_bq.py

```diff
@@ -0,0 +1,276 @@
+
+    method = LinSquareTransformRBFIsoGaussMeasureBayesianQuadrature(base_gp=model, X=X_train, Y=-Y_train,
+                                                                         bound=0)
+    assert_array_equal(method.alpha, .5)
+
+
+def test_square_transform_bq_model_invalid_bound():
+    with pytest.raises(ValueError):
+        X = np.array([[-2], [0], [3]])
+        Y = np.array([[4], [0], [9]])
+        dim = X.shape[1]
+        bound = 1  # intentionally incorrect
+
+        gpy_model = GPy.models.GPRegression(X=X, Y=Y,
+                                    kernel=GPy.kern.RBF(input_dim=dim,
+                                                        lengthscale=1., variance=1.))
+        measure = IsotropicGaussianMeasure(mean=np.ones(dim), variance=1.)
+
+        emukit_qrbf = QuadratureRBFIsoGaussMeasure(rbf_kernel=RBFGPy(gpy_model.kern),
+                            measure=measure)
+
+        emukit_model = BaseGaussianProcessGPy(kern=emukit_qrbf, gpy_model=gpy_model)
+
+        #error
+        emukit_method = LinSquareTransformRBFIsoGaussMeasureBayesianQuadrature(base_gp=emukit_model, X=X, Y=Y,
+                                                                                    bound=bound)
+
+
+
+def test_square_transform_bq_model_alpha_tuning():
+    X = np.array([[-2], [1], [3]])
+    Y = np.array([[-4.1], [-1.1], [-9.1]])
+    dim = X.shape[1]
+    bound = 0
+
+    gpy_model = GPy.models.GPRegression(X=X, Y=Y, kernel=GPy.kern.RBF(input_dim=dim))
+    measure = IsotropicGaussianMeasure(mean=np.ones(dim), variance=1.)
+    emukit_qrbf = QuadratureRBFIsoGaussMeasure(rbf_kernel=RBFGPy(gpy_model.kern), measure=measure)
+    emukit_model = BaseGaussianProcessGPy(kern=emukit_qrbf, gpy_model=gpy_model)
+    emukit_method = LinSquareTransformRBFIsoGaussMeasureBayesianQuadrature(base_gp=emukit_model,
+                                                                    X=X, Y=Y, bound=bound)
+    assert emukit_method.alpha == 0.55
+
+    # no change
+    X = np.array([[-2], [1], [3], [5]])
+    Y = np.array([[-4.1], [-1.1], [-9.1], [-25.1]])
+
+    emukit_method.set_data(X, Y)
+    assert emukit_method.alpha == 0.55
+
+    # change
+    X = np.array([[-2], [1], [3], [5], [0]])
+    Y = np.array([[-4.1], [-1.1], [-9.1], [-25.1], [-.1]])
+    emukit_method.set_data(X, Y)
+    assert emukit_method.alpha == 0.05
+
+
+def test_square_transform_bq_integrate(square_transform_bq):
+    # to check the integral, we check if it lies in some confidence interval.
+    # these intervals were computed as follows: the mean square_transform_bq.predict (first argument) was integrated by
+    # simple random sampling with 1e7 samples, and the variance (second argument) with 2*1e4 samples. This was done 100
+    # times. The intervals show mean\pm 3 std of the 100 integrals obtained by sampling. There might be a very small
+    # chance the true integrals lies outside the specified intervals.
+    interval_mean = [-0.6308909408102297, -0.6283025018967014]
```

Comment:
As intermediate solution (there is an issue regarding this) we also check in the script that computes those intervals with a fixed seed.

Also, the test is for 1D? I think would be better if it were 2D, no due to some covariances, no?